Program.cs

```diff
@@ -30,7 +29,17 @@ static void Main(string[] args)
             SaveData();
             HandlingOfMenu();
         }
+        public static List<Clinic> GetClinics()
+        {
+            Clinic animal = new Clinic("ANIMAL", "aaa");
+            Clinic kaczor = new Clinic("KACZOR", "kkk");
+
+            List<Clinic> listOfClinic = new List<Clinic>();

```

Comment:
You don't have to declare and initial again listOfClinic becouse You have it in line 21. Similarly with listOfCats and list of dogs, just use them here.